### PR TITLE
Typo "** How small"→"**How small"

### PR DESCRIPTION
**Pull request by hyoshioka0128**
https://docs.microsoft.com/en-us/office365/enterprise/add-several-users-at-the-same-time

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Enterprise/add-several-users-at-the-same-time.md [hunk 1]
```diff
@@ -100,9 +100,9 @@ When you save the file, you'll probably get a prompt that some features in your
...
     
-- ** What languages can I use? ** When you create your spreadsheet, you can enter user data column labels in any language or characters, but you must not change the order of the labels, as shown in the sample. You can then make entries into the fields, using any language or characters, and save your file in a Unicode or UTF-8 format. 
+- **What languages can I use?** When you create your spreadsheet, you can enter user data column labels in any language or characters, but you must not change the order of the labels, as shown in the sample. You can then make entries into the fields, using any language or characters, and save your file in a Unicode or UTF-8 format. 
     
 - **What if I'm adding users from different countries or regions?** Create a separate spreadsheet for each area. You'll need to step through the Bulk add users wizard which each spreadsheet, giving a single location of all users included in the file that you're working with.
```
Comment (JohanFreelancer9) [inline Enterprise/add-several-users-at-the-same-time.md:107]:
```suggestion
- **What if I'm adding users from different countries or regions?** Create a separate spreadsheet for each area. You'll need to step through the Bulk add users wizard with each spreadsheet, giving a single location of all users included in the file that you're working with. 
```

**Comment by mypil** [comment]
@hyoshioka0128 - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@Sirkkuw - PR is ready for final review. There are some minor copy/edit changes that needs to be done but the OP did not merge the commit suggested by @JohanFreelancer9.

Thank you.

cc: @kenwith

**Pull request merged**